crates/iroha_data_model/src/transaction.rs

```diff
@@ -62,27 +64,20 @@ mod model {
     /// Wrapper for byte representation of [`Executable::Wasm`].
     ///
     /// Uses **base64** (de-)serialization format.
```

Comment:
Docs should be adjusted to reflect that (de)serialization format is no longer always base64.